lib/raft/raft.cpp

```
@@ -0,0 +1,999 @@
+    return m_config.m_id;
+}
+
+auto raft_node_grpc_client_t::ip() const -> ip_t
+{
+    return m_config.m_ip;
+}
+
+// -------------------------------
+// consensus_module_t
+// -------------------------------
+consensus_module_t::consensus_module_t(node_config_t                        nodeConfig,
```

Cppcheck (2.10-2)

[performance] Function parameter 'nodeConfig' should be passed by const reference.

(passedByValue)